client/api_object.go

```diff
@@ -793,16 +826,23 @@ func (c *client) FGetObjectResumable(ctx context.Context, bucketName, objectName
 		return err
 	}
 
-	segNum = startOffset / segmentSize
-	// Wait for the segments download finished
-	for offset := startOffset; offset < int64(meta.PayloadSize); offset += segmentSize {
+	segNum = startOffset / int64(opts.PartSize)
+	if isRange {
+		endOffset = rangeEnd
+	} else {
+		endOffset = int64(meta.GetPayloadSize()) - 1
+	}
+	log.Debug().Msg(fmt.Sprintf("get object resumeable begin segment Range: %s, startOffset: %d, endOffset:%d", opts.Range, startOffset, endOffset))
```

Comment:
remove debug log if  all the test finished

Reply:
It is not good for user experience,  we will remove almost  all the log after the mainnet. You can delete these logs before the next testnet release version

Reply:
Didn't we discuss earlier to add log for go-sdk？

Reply:
yes ,  it is not suitable to have too many detailed logs. Necessary logs can be kept. Testnet can currently keep debug logs .

Reply:
and is recommended to add err log  when err !=nil. which help locate the error,  not  the info log